jabbercat/models.py

```diff
@@ -748,13 +747,13 @@ class RosterFilterModel(Qt.QSortFilterProxyModel):
     def __init__(self, parent: Qt.QObject=None):
         super().__init__(parent)
 
-        self._tags_filter_model = None
-        self._tags_filter_set = None
-        self._tags_filter_set_connections = []
-        self._filter_by_text = None
+        self._tags_filter_model: Qt.QAbstractItemModel = None
+        self._tags_filter_set: CheckModelSet = None
+        self._tags_filter_set_connections: List[Signal] = []
+        self._filter_text_input: str = None
```

Comment:
I do not agree with the renaming of the variable here. The old name wasn’t great either, but if we’re going to bikeshed this, we should do it "right":

* ``filter_text_input`` suggests that it is filtering the text input, not the list based on the text input
* ``filter_by_text`` sounds like a command, not like a value

I think the problem is with the overload of *filter* as noun and verb and there’s no perfect solution. I prefer keeping it unchanged instead of changing it for the sake of change, especially given the rather poor test coverage of the UI.

Reply:
How about `text_predicate`? I'm not in love with `filter_text_input`, but at list it gives a clue that it's an input. `filter_by_text` OTOH made me think of a lambda/function at first.

Reply:
``text_predicate`` makes *me* think of a lambda function. I think there’s no winning move here, so I’d prefer if we could roll this back.

Reply:
OK